### PR TITLE
Add aditional text box in the checkout page

### PR DESCRIPTION
**Pull request by SMH110**

### CODE REVIEW

File: front-end/app/checkout.component/payment-processing.component.ts [hunk 1]
```diff
@@ -43,6 +43,7 @@ export class PaymentProcess implements OnInit, OnDestroy {
                                     email: buyerDetails.email,
                                     phone: buyerDetails.phone
                                 },
+                                note: buyerDetails.note,
```
Comment (massimocode) [inline front-end/app/checkout.component/payment-processing.component.ts]: The note is part of the order, not the buyer details. The buyer details will be persisted between orders, but the notes should not be. Maybe notes should be added on the basket screen considering they are to do with the basket?

Comment (SMH110) [inline front-end/app/checkout.component/payment-processing.component.ts]: I didn't have any idea about the note and where it should to be, so I visit the old website of the client and I saw note input in his checkout page and placeholder of the input is this : "Notes about your order, e.g. special notes for delivery."
therefore I added in the checkout page. 
Shall I put it in the basket page now?

Comment (massimocode) [inline front-end/app/checkout.component/payment-processing.component.ts]: Wait on this, I'll get back to you

Comment (massimocode) [inline front-end/app/checkout.component/payment-processing.component.ts]: The checkout page is fine but it should not persist order notes between orders.

**Pull request closed without merging**